src/sentry/runner/commands/devserver.py

```diff
@@ -22,6 +22,9 @@
     help="Start local styleguide web server on port 9001",
 )
 @click.option("--environment", default="development", help="The environment name.")
+@click.option(
+    "--skip", default=None, required=False, help="Names of daemons not to start (comma-delimited)"
+)
```

Comment:
"Skip" was just the shortest word I could think of but I'm open to a more descriptive name if you have suggestions.

Reply:
I'd do `--skip-daemons`, and hint in the help that the daemons are printed as "started xxx" in the logs. Unfortunately it doesn't seem to me at a glance that it would be easy to massage the code to simply print the list of daemons that will be executed at option parsing time.

Reply:
Hmm, actually, you can probably add a daemons map to module-level and reference the keys in the help. It'd be like

```py
{  "cron": ["sentry", "run", "cron", "--autoreload"] }
```

Then simply later `daemons += all_daemons["cron"]`.

Reply:
👍 to `--skip-daemons` and to referencing the daemon named in the help. It might make the code more readable to extract the daemons anyway.

Might be trouble if the commands need to change dynamically, but even that could be solved with some fairly simple objects or callables.